### PR TITLE
feat: ema update

### PR DESCRIPTION
**Pull request by BrendaHijmans**
Some specific features in data model for EMA, under EMA/molgenis.csv: 
 -Publications table now refers to resources (Extended resources ref), composite key with doi
-add Publications.citation text. 
-delete all other columns under Publications.
-edit Linked resources.other linked resources: now part of key.
-add Mappings.cdms other text field as part of key.

For general data model:
-restructure linkage, Linked resources section.
-add Data sources.prelinked bool
-add Data sources.linkage possibility description text
-add Studies.medicines studies brands

-add ontologies: INN.csv, ATC.csv
-fixes Diseases.csv
-add Worldwide to Countries.csv and edit order
-add some Observation targets
-fix Areas of information ds
-fix Data source types

Still needs:
- [x] Update staging areas
- [x] update documentation
- [x] update templates
- [x] update test data
- [x] update documentation
- [x] Ontologies: Medicine brand names
- [x] data migration - re-upload molgenis.csv to all schemas

### CODE REVIEW

File: data/datacatalogue/molgenis.csv [hunk 1]
```diff
@@ -111,9 +109,11 @@ Databanks,,multiple entries,bool,,,,,,,,,"Can there be multiple entries for a si
 RWE resources,,linkage,heading,,,,,,,,,Data linkage,,,
 Databanks,,has identifier,bool,,,,,,,,,Is there a unique identifier for a person in the data bank?,,Databanks,hasIdentifier
 Databanks,,identifier description,text,,,,,,,,,"Describe the variable that is used as a unique identifier for a person in the data bank? If the unique identifier is not at level of a person (for example hospital encounter), describe how this translated to an individual level",,Databanks,identifierDescription
-RWE resources,,linkage description,text,,,,,,,,,Provide a high-level description of the linkages that are either: currently available between data sources in the data source (when pre-linked = yes); linkages that are possible when using the data source,EMA,,
```
Comment (mswertz) [inline data/datacatalogue/molgenis.csv:114]: this was a recommendation of Rosa, is it moved elsewhere?

Comment (mswertz) [reply data/datacatalogue/molgenis.csv:114]: Ah I see, a line below :-)

File: data/scripts/molgenis-model-update/requirements.txt [hunk 2]
```diff
@@ -1,12 +1,14 @@
 certifi==2023.7.22
 charset-normalizer==2.1.1
 idna==3.4
-numpy==1.26.2
-pandas==2.1.2
+numpy
```
Comment (mswertz) [inline data/scripts/molgenis-model-update/requirements.txt:4]: huh, you always want to use latest? I am okay with this but this might be not reproducible?

**Review by mswertz — COMMENTED** [review]
looks good. I would like another data manager to review and approve and then it can be merged.

File: data/scripts/molgenis-model-update/requirements.txt (hunk 2 above)
Comment (BrendaHijmans) [reply data/scripts/molgenis-model-update/requirements.txt:4]: true, I am changing this back

File: data/datacatalogue/stagingRWE/molgenis.csv
Comment (YpeZ) [inline data/datacatalogue/stagingRWE/molgenis.csv:1]: image
Got this error when trying to upload the stagingRWE/molgenis.csv to an empty staging area on the PR's dev server

File: data/datacatalogue/stagingRWE/molgenis.csv [hunk 3]
```diff
@@ -112,12 +112,13 @@ RWE resources,,quality validation methods,text,,,,,,,,,What methods or processes
...
 RWE resources,,standards,heading,,,,,,,,,Use of standard data models and ontologies,
+RWE resources,,cdms,ref_array,,,,Mappings,,source,,,Common data models used or ETL-ed to by this data source,
 RWE resources,,cdms other,text,,,,,,,,,"If not in list above, give the name of cdm(s) used by this data source",EMA
 Databanks,,ETL standard vocabularies,ontology_array,,,CatalogueOntologies,Vocabularies,,,,,"Are data mapped to standardised vocabularies during ETL to the CDM? If yes, what vocabularies are used for events, such as diagnoses?",
 Databanks,,ETL standard vocabularies other,text,,,,,,,,,"If other, what other vocabularies are used?",
 Data resources,,information,heading,,,,,,,,,Other information,
-Data resources,,design paper,ref_array,,,,Publications,,,,,Publication(s) that describe(s) the design of this resource,
-Extended resources,,publications,ref_array,,,,Publications,,,,,Other publication(s) about this resource,EMA
+Data resources,,design paper,refback,,,,Publications,,resource,,,Publication(s) that describe(s) the design of this resource,
+Extended resources,,publications,refback,,,,Publications,,resource,,,Other publication(s) about this resource,EMA
```
Comment (YpeZ) [inline data/datacatalogue/stagingRWE/molgenis.csv:121]: Then 

> Publications,,resources,**refback**,,,,

should be changed to 

> Publications,,resources,ref(_array?),,,,

?

File: data/datacatalogue/CatalogueOntologies/Countries.csv [hunk 4]
```diff
@@ -1,250 +1,251 @@
...
-Viet Nam,VN,241,,,
-Virgin Islands (British),VG,242,,,
-Virgin Islands (U.S.),VI,243,,,
-Wallis and Futuna,WF,244,,,
-Western Sahara*,EH,245,,,
-Yemen,YE,246,,,
-Zambia,ZM,247,,,
-Zimbabwe,ZW,248,,,
+order,name,code,definition,parent,ontologyTermURI
+1,Worldwide,,,,
```
Comment (YpeZ) [inline data/datacatalogue/CatalogueOntologies/Countries.csv:2]: Shouldn't this option be in _Regions_?

**Review by YpeZ — COMMENTED** [review]
Some comments on the big ontologies.

stagingRWE fails to upload due to conflicting _refBack_ columns.

Didn't test the Python scripts, but I'm fine if they work for you. Could be moved to new scripts repo?

File: data/datacatalogue/CatalogueOntologies/ATC.csv
[the remark this answers is not in the document]
Comment (BrendaHijmans) [reply data/datacatalogue/CatalogueOntologies/ATC.csv:1]: This make up of the name was a request from EMA. This is how they extracted their data from their old database. Some substances have multiple id's/entries in the ATC ontology, e.g. epinephrine, so we cannot just use the name, but have to concatenate with the id.

File: data/datacatalogue/CatalogueOntologies/INN.csv
[the remark this answers is not in the document]
Comment (BrendaHijmans) [reply data/datacatalogue/CatalogueOntologies/INN.csv:1]: This make up was requested by EMA. I can add the more readable lowercase in label, but it is still the wrong spelling. Some characters need to be upper case, such as the C in 13C.

File: data/datacatalogue/CatalogueOntologies/Countries.csv (hunk 4 above)
Comment (BrendaHijmans) [reply data/datacatalogue/CatalogueOntologies/Countries.csv:2]: Regions are now more considered to be smaller portions of countries. What UMCG wants to say here is: subjects/data may come from all countries in the world.

**Comment by YpeZ** [comment]
image
Getting this error when trying to upload schema EMA_3.8/molgenis.csv in an empty staging area on the PR's preview server

**Comment by BrendaHijmans** [comment]
> image Getting this error when trying to upload schema EMA_3.8/molgenis.csv in an empty staging area on the PR's preview serve, 

I will fix this.

File: data/datacatalogue/CatalogueOntologies/Study topics.csv
[the remark this answers is not in the document]
Comment (BrendaHijmans) [reply data/datacatalogue/CatalogueOntologies/Study topics.csv:1]: there was a nbsp there at first, changed to a normal space

File: data/datacatalogue/stagingCohortsUMCG/molgenis.csv
[the remark this answers is not in the document]
Comment (BrendaHijmans) [reply data/datacatalogue/stagingCohortsUMCG/molgenis.csv:1]: should be fixed now, 'resource' was under refTable instead of refLink

**Review by YpeZ — APPROVED** [review]

**Pull request merged**